genomicFileHandler/genomic_file_handlers.py

```diff
@@ -106,43 +91,40 @@ def get_sample_value(self, variable, idx=0):
 
 class Bam_header:
```

Comment:
Do you need this class? `pysam.AlignmentFile.header` should give you the BAM header information.